src/components/details-view/accordion-list-header.js

```diff
@@ -1,19 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormattedNumber, injectIntl, intlShape } from 'react-intl';
+import { FormattedMessage, FormattedNumber, injectIntl, intlShape } from 'react-intl';
 import { KeyValue } from '@folio/stripes-components';
 import AccordionHeader from '@folio/stripes-components/lib/Accordion/headers/DefaultAccordionHeader';
 import styles from './accordion-list-header.css';
 
 function AccordionListHeader(props) {
   let { intl } = props;
+  // RM API does not return exact number of results when count is over 10K
+  // For title lists, resultsLength of 10000 indicates this.
+  // For other lists (package and vendor) resultsLength of 10001 indicates this.
```

Comment:
a very minor nitpick - could we please update the comment to say "provider" instead of "vendor"?

Reply:
Update for correct terminology -- thanks